### PR TITLE
watsonx.ai L4 Lab - LLM Application Building Blocks

### PR DESCRIPTION
**Pull request by maurelya**
Relates to **[[issue link](https://github.com/orgs/ibm-build-lab/projects/4?pane=issue&itemId=50324820)]**

Content changes:

- Added markdown files for watsonx.ai 109 lab http://localhost:8000/en/watsonx/watsonxai/109
- Updated README.md http://localhost:8000/en/watsonx/watsonxai
- updated classification notebooks
-

### CODE REVIEW

File: content/Watsonx/WatsonxAI/109.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />
+
+# Large language model application building blocks
+
+This hands-on exercise will show you how to integrate large language models (LLMs) with client applications. We will review several Python code samples which can be used as “building blocks” for an LLM application.
+We will then use a simple UI prototype implemented with [Streamlit](https://streamlit.io/) to show how these building blocks can be invoked from client applications.
+
+Prompt engineering is just one of the steps in the process of integrating LLMs into business applications. It is outside the scope of this guide to provide an introduction to prompt engineering in watsonx.ai. For an introduction to prompt engineering in watsonx.ai, check out the [VEST watsonx.ai L3 Labs](/watsonx/watsonxai)
+
+## Prerequisites
```
Comment (jbukuts) [inline content/Watsonx/WatsonxAI/109.md:16]: One addition that might be needed is a requirements.txt or some code block mentioning all dependencies to install with tested versions. For reference here's what I used:

```
load_dotenv==0.1.0
ibm_watson_machine_learning==1.0.345
streamlit==1.31.0
plotly==5.18.0
```

**Review by jbukuts — APPROVED** [review]
Looks good. One minor thing might be to move the 109 folder with the scripts, CSVs, notebooks, and such into the `files` folder. 

Also if we want to number L4 starting in the 200s we might need to renumber but we can iterate on that once all the content is in.

File: content/Watsonx/WatsonxAI/109.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+     - The **main()** function specifies the model parameters and invokes the other functions. The function prints LLM output to the terminal. 
+    
+        ![main](./images/109/main.png)
+
+     - Lastly, each script contains at least one function that can be called by an external Python module. For example, in _use_case_generate.py_, the function name is **generate**.
+
+2. Prior to running the scripts, create a _.env_ file in the root directory of your project and add your IBM Cloud API key and watsonx project id. 
+
+      > For more information, see [Creating an IBM Cloud API key](https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui#create_user_key) and [Looking up your watsonx project ID](https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/fm-project-id.html?context=wx).
```
[the remark this answers is not in the document]
Comment (maurelya) [reply content/Watsonx/WatsonxAI/109.md]: Here is the follow up item: https://github.com/ibm-build-lab/VAD-VAR-Workshop/issues/315

File: content/Watsonx/WatsonxAI/109.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+3. Open a terminal and navigate to the scripts's location. Run each script. For example you can run one of the scripts with the following command: 
+
+      ``` bash
+         python ./use_case_generate.py
```
[the remark this answers is not in the document]
Comment (maurelya) [reply content/Watsonx/WatsonxAI/109.md]: One of the quiz questions asks `When does it make the most sense to make
prompting calls to watsonx.ai LLMs from Python scripts, as
opposed to Jupyter notebooks?` For the attendees to answer this question, we have to maintain the Python scripts and not convert them to Jupyter notebooks.

[the remark this answers is not in the document]
Comment (krab7191) [reply content/Watsonx/WatsonxAI/109.md]: Sounds good. I just don't want to run into the issue where attendees have issues. Hopefully that won't happen again as we go forward with a more technical audience.

**Comment by krab7191** [comment]
> Looks good. One minor thing might be to move the 109 folder with the scripts, CSVs, notebooks, and such into the `files` folder.
> 
> Also if we want to number L4 starting in the 200s we might need to renumber but we can iterate on that once all the content is in.

Agreed, ideally I would like to have "L3" and "L4" sections, but we might as well hold off since it seems we'll be doing a site-wide redo

**Pull request merged**